src/hooks/useScanApi.ts

```diff
@@ -242,19 +242,9 @@ const useGetScanJobsApi = (onAddAlert: (alert: AlertProps) => void) => {
     []
   );
 
-  const callbackSuccess = useCallback(
-    (response: AxiosResponse<ScanJobsResponse>) => {
-      onAddAlert({
-        title: t('toast-notifications.description', {
-          context: 'scan-jobs_fetched',
```

Comment:
Should that string be also removed from `public/locales/en.json`? As far as I can tell, this is the only place it is used (but I might be wrong!).

Reply:
Yes, agreed! I added the removal of that string to the commit.